modules/javafx.graphics/src/main/native-glass/monocle/egl/egl_ext.h

```diff
@@ -0,0 +1,16 @@
+#include <jni.h>
```

Comment:
This needs a standard copyright header.

Also should it have a guard to prevent including it more than once?

Reply:
correct. fixed.